lib/GLPI/Agent/SNMP/MibSupport/BrotherNetConfig.pm

```diff
@@ -77,6 +81,37 @@ sub getModel {
     return $model;
 }
 
+sub updatePortIfType {
+    my ($self) = @_;
+
+    my $device = $self->device
+        or return;
+
+    # Get list of device ports
+    my $ports = $device->{PORTS}->{PORT};
+
+    # Get list of device ports types (lan(1)/wirelesslan(2))
+    my $brMultiIFType = $self->walk(brMultiIFType)
+        or return;
+
+    # Get list of device ports names
+    my $brMultiIFNodeType = $self->walk(brMultiIFNodeType)
+        or return;
+
+    foreach my $index (keys %{$brMultiIFType}) {
+        foreach my $port (keys %{$ports}) {
+            next unless defined($ports->{$port}->{IFNAME}) && defined($brMultiIFNodeType->{$index});
+            if ($ports->{$port}->{IFNAME} eq $brMultiIFNodeType->{$index}) {
```

Comment:
Can you please update the test here to use `getCanonicalString()` ? This will guaranty I can use your snmpwalk output in our internal tests.
```suggestion
            if ($ports->{$port}->{IFNAME} eq getCanonicalString($brMultiIFNodeType->{$index})) {
```